### PR TITLE
Provide utility class to provide portal user info

### PR DESCRIPTION
**Pull request by sven1103**
This PR adds a utility class, that will provide an optional `PortalUser` with user information, so that the app can list user based project information.

### CODE REVIEW

**Comment by sven1103** [comment]
[E01.05: Create class that holds logged-in user information](https://trello.com/c/1yEmluQm/827-e0105-create-class-that-holds-logged-in-user-information)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -10,11 +10,12 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
...
 **Added**
 
+* Provides a ``life.qbic.portal.sampletracking.system.Utils`` class, that provides the current logged in user
 * Github workflow that checks that the changelog has been updated
 * Created the project using cookietemple
 
 **Fixed**
 
-**Dependencies**
+**Dependencies*
```
Comment (KochTobi) [inline CHANGELOG.rst:19]:
```suggestion
**Dependencies**
```
Was this by accident?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -10,11 +10,12 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 **Added**
 
+* Provides a ``life.qbic.portal.sampletracking.system.Utils`` class, that provides the current logged in user
```
Comment (KochTobi) [inline CHANGELOG.rst:13]:
```suggestion
* Provides a ``life.qbic.portal.sampletracking.system.Utils`` class, that provides the current logged in user (`#21 <https://github.com/qbicsoftware/sample-tracking-status-overview/pull/21>`_)
```

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/system/Utils.groovy [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import com.vaadin.server.VaadinService
+import life.qbic.datamodel.dtos.portal.PortalUser
+import life.qbic.portal.utils.user.UserRelated
+
+/**
+ * <p>Utility class that provides access to some system context properties.</p>
+ *
+ * @since 1.0.0
+ */
+class Utils {
```
Comment (KochTobi) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/system/Utils.groovy:13]:
```suggestion
class SystemContext {
```
I would suggest renaming this class to give it meaning. Utils is pretty generic and as far as I understood it we use this to handle the system context?

**Review by KochTobi — CHANGES_REQUESTED** [review]
Looks good. I think there is a problem in the changelog. In addition to that I made a suggestion as to the naming of the class added in this PR.

**Comment by sven1103** [comment]
@KochTobi thanks a lot, addressed your requests

**Review by KochTobi — APPROVED** [review]

**Pull request merged**